lampo-common/src/conf.rs

```diff
@@ -97,10 +98,10 @@ impl LampoConf {
                 // so it is safe unwrap here otherwise we are hiding a bug
                 // and we must crash.
                 .or_else(|| root_path.strip_suffix(&suffix_without_slash))
-                .expect(&format!("path: {root_path} - network: {network}"))
+                .unwrap_or_else(|| panic!("path: {root_path} - network: {network}"))
```

Comment:
Why are we panicking here?

Reply:
Good question, because this should never fails